### ISSUE TITLE
LWRP: ComputeBuffer doesn't render in camera

### ISSUE THREAD

**Issue by gameanimation3d** [keijiro/Pcx#34]
With the New LWRP. 
The Pointcloud in ComputeBuffer Mode doesn't render in the Camera. Only in Editor the Pointcloud gets correctly drawn.
If i switch back to builtin Renderpipeline everything works fine.

Mesh Rendering in LWRP works. But doesn't apply an Point Size.

Unity Version: 2019.2.8f1
Pcx Version: Release Version v1.2

**Comment by keijiro** [comment]
> Only in Editor the Pointcloud gets correctly drawn.

So, you meant that this issue only reproduces on a built app?

Which platform are you using? Windows, macOS or mobile platform?

**Comment by gameanimation3d** [comment]
> So, you meant that this issue only reproduces on a built app?

No. The pointcloud gets only drawn in the Editor View. In the Game View in Editor it doesn't show up.
I didn't tested it it an Build app

> Which platform are you using? Windows, macOS or mobile platform?

As OS Windows 10 64bit - Build Settings are for Windows

**Comment by bjornsyse** [comment]
Have the same issue. Even get a nullReferenceExection on play. Something with the camera? 

```
NullReferenceException: Object reference not set to an instance of an object
Pcx.PointCloudRenderer.OnRenderObject () (at Assets/Pcx/Runtime/PointCloudRenderer.cs:88)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)

```

**Comment by ahwemorela** [comment]
Did you manage to solve the problem?

**Comment by StrikeAgainst** [comment]
Struggling with the same issue in HDRP. Is there no solution yet?

**Comment by StrikeAgainst** [comment]
Update as per [this thread](https://forum.unity.com/threads/running-a-geometry-shader-with-computebuffer-in-hdrp.1465718/) I opened on the Unity forums: `OnRenderObject` does not seem to get triggered with HDRP (possibly LWRP and any other scriptable pipeline as well), thus you have to attach the rendering process to the manager yourself. Worked fine for me after doing so.
